File: Eclipse/VersaoComBanco/ProjetoFinalDm102/src/br/inatel/mobile/pos/dm102/trabalhofinal/model/Cliente.java
package br.inatel.mobile.pos.dm102.trabalhofinal.model;

public class Cliente {

	private String nome;
	private String endereco;
	private String telefone;

	public Cliente(String nome, String endereco, String telefone) {
		this.nome = nome;
		this.endereco = endereco;
		this.telefone = telefone;
	}

	public String getNome() {
		return nome;
	}

	public String getEndereco() {
		return endereco;
	}

	public String getTelefone() {
		return telefone;
	}

	@Override
	public String toString() {
		return "Nome: " + nome + " -- Endereco: " + endereco + " -- Telefone: " + telefone;
	}

}

File: VisualStudio/ProjetoFinalDm102/ProjetoFinalDm102/Cliente.cs
using System.Collections.Generic;

namespace ProjetoFinalDm102
{
    class Cliente
    {
        private string nome;
        private string endereco;
        private string telefone;

        private List<Atendimento> atendimentos = new List<Atendimento>();

        public Cliente(string nome, string endereco, string telefone)
        {
            this.nome = nome;
            this.endereco = endereco;
            this.telefone = telefone;
        }

        public string GetNome()
        {
            return nome;
        }

        public string GetEndereco()
        {
            return endereco;
        }

        public string GetTelefone()
        {
            return telefone;
        }

        public List<Atendimento> GetAtendimentos()
        {
            return atendimentos;
        }

        public void SalvarAtendimento(string descricao)
        {
            Atendimento atendimento = new Atendimento(descricao);
            atendimentos.Add(atendimento);
        }

        public override string ToString()
        {
            return "Nome: " + nome + " - Endereco: " + endereco + " - Telefone: " + telefone + " - Atendimentos: " + atendimentos.Count; 
        }

    }
}
